ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -18,12 +18,51 @@ periodics:
     labels:
       master: ci.openshift.redhat.com
     name: azure_build_node_image_centos
-- agent: jenkins
+- agent: kubernetes
```

Comment:
Add it in parallel so we can first ensure it is working and then we can remove the Jenkins base build.

Reply:
good point

Reply:
Run your job as a parent w/o any children.

Reply:
Also use `azure-build-base-image-rhel` as a name for the job.

Reply:
copy that